src/redfishpower/redfishpower.c

```diff
@@ -121,10 +121,13 @@ struct powermsg {
      * timeout - when the overall power command times out
      *
```

Comment:
Commit message should say what hardware the observations were based on and perhaps have a bit more detail about the observations themselves?  Without that context, when some hardware that behaves differently comes a long, we can't make an informed decision about how to move forward.